parser/parser.py

```diff
@@ -184,6 +185,25 @@ def allToJSON(self, toConvert):
             self.toJSON(fileName)
         self.getAllSongs()
 
+    def parseSongTags(self):
+        """
+        Update ALL_SONGS.json with song tag metadata
+        """
+        songsToTags = {}
+        for fileName in sorted(os.listdir(TAG_FOLDER)):
```

Comment:
Let’s check that the file name ends with `.txt`, so we don’t end up with tags for `.txt~` or `.txt.orig` or `.DS_Store` or whatever.